marketplace/deployer_util/provision.py

```diff
@@ -99,6 +100,10 @@ def process(schema, values, deployer_image, deployer_entrypoint, version_repo,
     elif prop.xtype == config_helper.XTYPE_INGRESS_AVAILABLE:
       # TODO: Really populate this value.
```

Comment:
Since you are touching this file, can you link to #360

TODO(#360): Really populate this value.